content-tree.d.ts

```diff
@@ -211,17 +211,27 @@ export declare namespace ContentTree {
         credit: string;
         picture?: ImageSetPicture;
     }
+    type TableColumnSettings = {
+        hideOnMobile: boolean;
```

Comment:
**Question:** I'm not actually sure how to figure `hideOnMobile` out, the only thing I can find in o-table is about about **[responsive-flat](https://github.com/Financial-Times/origami/blob/813c8e46d9e6e423fad3a1322d043edc6422cd69/components/o-table/src/scss/_responsive-flat.scss#L3)**. Is this the right thing?

Reply:
hmm. that's a good question. i'm actually not even sure where it came from